K2Bridge.Tests.UnitTests/Visitors/MatchPhraseVisitorTests.cs

```diff
@@ -19,6 +20,30 @@ public string MatchPhraseVisit_WithValidClause_ReturnsEquals()
             return VisitQuery(matchPhraseClause);
         }
 
+        [TestCase(ExpectedResult = "MyField == \"3\"")]
+        public string MatchPhraseVisit_WithValidIntClause_ReturnsEquals()
+        {
+            var matchPhraseClause = CreateMatchPhraseClause("MyField", 3);
+
+            return VisitQuery(matchPhraseClause);
+        }
+
+        [TestCase(ExpectedResult = "MyField == \"3.5\"")]
```

Comment:
is this correct? should 3.5 treated as string?

Reply:
It does work, but now I changed it to be more idiomatic